style.css

```diff
@@ -80,3 +83,172 @@ nav #myLogo {
   font-size: 18px;
   color: #6070ff;
 }
+
+.grid-box {
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #e5e5e5;
+  border-radius: 8px;
+  width: 100%;
+  gap: 88px;
+  padding: 104px 0;
+}
+
+.icon {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+  padding: 0;
+}
+
+.box {
+  background-color: #fff;
+  gap: 10px;
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  width: min(327px, 80vw);
+  border: 2px solid #dfe1e6;
+}
+
+.banner {
+  border-radius: 8px;
+  width: min(327px, 80vw);
+}
+
+input {
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  background-color: #fff;
+  border-color: #6070ff;
+  border-radius: 8px;
+  padding: 8px 5px;
+  display: flex;
+  width: min(127px, 80vw);
+}
+
+h1 {
+  font-size: 32px;
+  line-height: 44px;
+}
+
+p {
+  text-align: left;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+@media (max-width: 330px) {
+  ul {
+    flex-wrap: wrap;
+  }
+}
+
+.tag {
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  color: #6070ff;
+}
+
+.about {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 49px;
+  flex-wrap: wrap;
+}
+
+.about-header {
+  line-height: 52px;
+  font-size: 40px;
+}
+
+.resume {
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  background-color: #fff;
+  border-color: #6070ff;
+  padding: 4px 14px 4px 14px;
+  display: flex;
+  border-radius: 8px;
+}
+
+.drop {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.drop-text {
+  font-size: 15px;
+  line-height: 20px;
+  padding-left: 16px;
+  padding-right: 150px;
+}
+
+.drop-box {
+  display: flex;
+  flex-direction: column;
+}
+
+.drop-li {
+  display: flex;
+  background-color: #f7f7f9;
+  padding: 12px 12px;
+  border-radius: 8px;
+}
+
+.lang {
+  display: flex;
+  padding: 0;
+  justify-content: space-between;
+}
+
+.union {
+  height: 10px;
+  align-self: center;
+}
+
+.union-down {
+  height: 18px;
+  align-self: center;
+}
+
+.dropdown {
+  display: flex;
+  flex-direction: column;
+  padding: 0 49px;
+  flex-wrap: wrap;
+}
```

Comment:
- [ ] Kindly apply the recommendations I made on your project cards to fix the appearance of your `about section` too. I know you can do this, champ 💪 

**Your Design**
![image](https://user-images.githubusercontent.com/99715640/212442510-a89e922c-77b4-4208-b198-d9d9d14fb03b.png)